app/src/main/res/values/styles.xml

```diff
@@ -599,4 +607,8 @@
         <item name="strokeColor">?attr/paper_color</item>
     </style>
 
+    <style name="MaterialDatePickerStyle" parent="ThemeOverlay.Material3.MaterialCalendar">
```

Comment:
The date picker looks a bit incorrect in Dark mode (dark text on dark background.).

Reply:
Good catch! Updated based on [this document](https://github.com/material-components/material-components-android/blob/master/docs/components/DatePicker.md).

Reply:
I also updated the text in the single selection text in the dialog (Donor vs. Not a donor) to use a similar setting, based on [this document too](https://github.com/material-components/material-components-android/blob/master/docs/components/Dialog.md).